### PR TITLE
Revamp Lobby llayout

### PR DESCRIPTION
**Pull request by enochtsang**
image

### CODE REVIEW

File: frontend/src/modules/lobby/components/PlayerList.vue [hunk 1]
```diff
@@ -1,29 +1,74 @@
...
-    }
+        ChangeNicknameModal,
+    },
+    methods: {
+        isCurrentPlayer: function (playerUuid) {
+            console.log(
+                "this player uuid : " +
+                    store.getters["application/getPlayerUuid"]
+            );
+            console.log("other player uuid : " + playerUuid);
```
Comment (mpminardi) [inline frontend/src/modules/lobby/components/PlayerList.vue:40]: Are these log statements needed?

File: frontend/src/modules/lobby/components/InvitationLink.vue [hunk 2]
```diff
@@ -48,12 +44,32 @@ export default {
...
+.invitation-link {
+    display: grid;
+    grid-template-columns: auto 300px auto;
+    grid-template-rows: 38px;
+    border-radius: 10px;
+    border: 1px solid black;
 }
 
-#invitation-link {
+.invitation-link__title {
```
Comment (mpminardi) [inline frontend/src/modules/lobby/components/InvitationLink.vue:55]: Super nit-picky, but do any of these actually need to be classes?

Comment (enochtsang) [reply frontend/src/modules/lobby/components/InvitationLink.vue:55]: I hear mostly that the by default you should use classes, and ids should be reserved for special cases, not the other way around.

File: frontend/src/modules/lobby/components/PlayerList.vue (hunk 1 above)
Comment (enochtsang) [reply frontend/src/modules/lobby/components/PlayerList.vue:40]: Removed

**Pull request merged**